tuner/tuner/libtuner.py

```diff
@@ -70,6 +75,7 @@ class CandidateTracker:
     dispatch_config_path: Optional[Path] = None
     configuration: Optional[candidate_gen.iree_codegen.CompilationInfoAttr] = None
     compilation_successful: Optional[bool] = None
+    compiled_vmfb_path: Optional[Path] = None
```

Comment:
Is this vmfb of the model or the dispatch?

Reply:
This is used for both. When you run the compile() function, it will populate this path. I think it is fine to overwrite things like this, because eventually I want to move the candidate_trackers to be private to some tuner object that can save state between tuning phases. Then we can ensure that the paths are filled as expected, and the tuning client doesn't have to touch these candidate trackers.